src/components/JourneysList.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect } from 'react';
+import { Journey } from '../models/Journey';
+
+type JourneyListProps = {
+    journeyData: Journey[];
+}
+const JourneysList: React.FC<JourneyListProps> = ({ journeyData }) => {
+    useEffect(() => {
+        console.log(journeyData);
+    }, [journeyData]);
+
+    return (
+        <div>
+            <tbody>
+                <tr>
+                    <th>Depart Time</th>
+                    <th>Arrival Time</th>
+                </tr>
+                {journeyData.map(( item, index ) => (
```

Comment:
Specify what the item is here with something more descriptive please

Reply:
changed it to journey